### DISCUSSION TITLE
Disable ssl in Keycloak 20.0.1

### DISCUSSION THREAD

**Discussion by rcbandit111** [keycloak/keycloak#15857] [Archive]
I'm trying to disable https:

I started the sever using `bin/kc.sh start-dev`

And then I tried to disable ssl using:

    root@server:/opt/keycloak#
    root@server:/opt/keycloak# /opt/keycloak/bin/kcadm.sh config credentials --server http://localhost:8080/auth --realm master --user admin –-password admin
    Logging into http://locahost:8080/auth as user admin of realm master
    Enter password: *****
    null [RESTEASY003210: Could not find resource for full path: http://localhost:8080/auth/realms/master/protocol/openid-connect/token]
    root@server:/opt/keycloak#

Do you know what is the proper way to execute the command?

**Comment by GerDner** [comment]
remove `/auth` from your url. It's not needed/supported anymore.
